### PR TITLE
Harshit/DF-80/Add discription to freelancer

### PR DESCRIPTION
**Pull request by harshitsharma7017**
Add description in freelancer model not more that 500 character count

### CODE REVIEW

File: src/models/freelancer.entity.ts [hunk 1]
```diff
@@ -179,6 +180,11 @@ const FreelancerSchema: Schema = new Schema(
       type: Date,
       required: false,
     },
+    discription: {
```
Comment (Akhil-chaudhary) [inline src/models/freelancer.entity.ts:183]: typo in description

Comment (harshitsharma7017) [reply src/models/freelancer.entity.ts:183]: couldn't understand, please give brief

Comment (Akhil-chaudhary) [reply src/models/freelancer.entity.ts:183]: 'discription' does this spelling looks right to you?

Comment (harshitsharma7017) [reply src/models/freelancer.entity.ts:183]: sorry sir

File: src/models/freelancer.entity.ts [hunk 2]
```diff
@@ -30,6 +30,7 @@ export interface IFreelancer extends Document {
   email: string;
   phone: string;
   dob?: Date;
+  discription?: string;
   professionalInfo?: Map<
```
Comment (Akhil-chaudhary) [inline src/models/freelancer.entity.ts:34]: fix it at everyplace

File: src/services/freelancer.service.ts [hunk 3]
```diff
@@ -208,6 +211,9 @@ export class FreelancerService extends BaseService {
       { _id: freelancer_id },
       freelancer,
     );
+    if (data.discription && data.discription.length > 500) {
+      throw new Error('Description cannot exceed 500 characters.');
+    }
```
Comment (Akhil-chaudhary) [inline src/services/freelancer.service.ts:216]: here

File: src/services/freelancer.service.ts [hunk 4]
```diff
@@ -132,6 +132,9 @@ export class FreelancerService extends BaseService {
       // });
       const userObj = { ...freelancer, password: "" };
       const data: any = await this.FreelancerDAO.createFreelancer(userObj);
+      if (data.discription && data.discription.length > 500) {
+        throw new Error('Description cannot exceed 500 characters.');
+      }
```
Comment (Akhil-chaudhary) [inline src/services/freelancer.service.ts:137]: here

File: src/models/freelancer.entity.ts (hunk 1 above)
Comment (Akhil-chaudhary) [reply src/models/freelancer.entity.ts]: No need to apologize; typos are completely normal.

**Pull request merged**